### PR TITLE
[SPARK-24560][CORE][MESOS] Fix some getTimeAsMs as getTimeAsSeconds

### PR DESCRIPTION
**Pull request by xueyumusic**
## What changes were proposed in this pull request?

This PR replaces some "getTimeAsMs" with "getTimeAsSeconds". This will return a wrong value when the user specifies a value without a time unit.

## How was this patch tested?
manual test

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/executor/Executor.scala [hunk 1]
```diff
@@ -613,7 +613,7 @@ private[spark] class Executor(
     private[this] val taskId: Long = taskRunner.taskId
 
     private[this] val killPollingIntervalMs: Long =
-      conf.getTimeAsMs("spark.task.reaper.pollingInterval", "10s")
+      conf.getTimeAsSeconds("spark.task.reaper.pollingInterval", "10s") * 1000L
```
Comment (maropu) [inline core/src/main/scala/org/apache/spark/executor/Executor.scala]: plz use `DurationConversions`.

**Comment by maropu** [comment]
Can you add tests in `SparkConfSuite`?

File: core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala [hunk 2]
```diff
@@ -354,7 +354,7 @@ private[spark] abstract class BasePythonRunner[IN, OUT](
     extends Thread(s"Worker Monitor for $pythonExec") {
 
     /** How long to wait before killing the python worker if a task cannot be interrupted. */
-    private val taskKillTimeout = env.conf.getTimeAsMs("spark.python.task.killTimeout", "2s")
+    private val taskKillTimeout = env.conf.getTimeAsSeconds("spark.python.task.killTimeout", "2s") * 1000L
```
Comment (maropu) [inline core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala]: Nit: Probably, we'd be better to add `Ms` in the suffix, => `taskKillTimeoutMs`

**Comment by xueyumusic** [comment]
I have made some modification, @maropu please review the code, thanks

**Comment by ueshin** [comment]
ok to test.

**Comment by maropu** [comment]
cc @jiangxb1987

**Comment by maropu** [comment]
@xueyumusic Can you fix the error first?

**Comment by maropu** [comment]
retest this please

File: core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala [hunk 3]
```diff
@@ -354,7 +355,8 @@ private[spark] abstract class BasePythonRunner[IN, OUT](
     extends Thread(s"Worker Monitor for $pythonExec") {
 
     /** How long to wait before killing the python worker if a task cannot be interrupted. */
-    private val taskKillTimeout = env.conf.getTimeAsMs("spark.python.task.killTimeout", "2s")
+    private val taskKillTimeoutMs = env.conf.getTimeAsSeconds("spark.python.task.killTimeout",
```
Comment (jiangxb1987) [inline core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala]: The conf is not documented, but I think it's designed to accept values like 1.5s
cc @zsxwing

File: core/src/main/scala/org/apache/spark/deploy/worker/DriverRunner.scala [hunk 4]
```diff
@@ -58,7 +59,7 @@ private[deploy] class DriverRunner(
 
   // Timeout to wait for when trying to terminate a driver.
   private val DRIVER_TERMINATE_TIMEOUT_MS =
-    conf.getTimeAsMs("spark.worker.driverTerminateTimeout", "10s")
+    conf.getTimeAsSeconds("spark.worker.driverTerminateTimeout", "10s").seconds.toMillis
```
Comment (jiangxb1987) [inline core/src/main/scala/org/apache/spark/deploy/worker/DriverRunner.scala]: ditto

File: core/src/main/scala/org/apache/spark/executor/Executor.scala [hunk 5]
```diff
@@ -613,7 +614,7 @@ private[spark] class Executor(
     private[this] val taskId: Long = taskRunner.taskId
 
     private[this] val killPollingIntervalMs: Long =
-      conf.getTimeAsMs("spark.task.reaper.pollingInterval", "10s")
+      conf.getTimeAsSeconds("spark.task.reaper.pollingInterval", "10s").seconds.toMillis
```
Comment (jiangxb1987) [inline core/src/main/scala/org/apache/spark/executor/Executor.scala]: Actually I believe this is change things from right to wrong. If you want to use a second value then you shall use sth like `1s`.

File: core/src/main/scala/org/apache/spark/ui/ConsoleProgressBar.scala [hunk 6]
```diff
@@ -34,7 +34,7 @@ private[spark] class ConsoleProgressBar(sc: SparkContext) extends Logging {
   private val CR = '\r'
   // Update period of progress bar, in milliseconds
   private val updatePeriodMSec =
-    sc.getConf.getTimeAsMs("spark.ui.consoleProgress.update.interval", "200")
+    sc.getConf.getTimeAsMs("spark.ui.consoleProgress.update.interval", "200ms")
```
Comment (jiangxb1987) [inline core/src/main/scala/org/apache/spark/ui/ConsoleProgressBar.scala]: We shall document this config.

**Comment by jiangxb1987** [comment]
Overall I don't think the current logic shall be modified. However, it shall be useful to document some the configs mentioned in this PR.

**Comment by xueyumusic** [comment]
I see, thanks for your review and guidance, @jiangxb1987 @maropu , I will try to add related config to doc and close this PR, thank you

**Pull request closed without merging**